workshops/maze/README.md

```diff
@@ -0,0 +1,113 @@
+# Friendly Maze Game
+
+## 1. Demo
+
+Your mission is to build your own version of
+[The Scary Maze Game](http://www.playscarymazegame.net/play-scary-maze-game/).
+
+Navigate your mouse through the maze without touching the walls:
+
+![](img/demo.gif)
+
+<a href="http://output.jsbin.com/xivifu/12" target="_blank_">![](img/bttn_play_now.png)</a>
+<a href="http://jsbin.com/xivifu/12/edit?js,output" target="_blank_">![](img/bttn_view_code.png)</a>
+
+## 2. How to Use This Tutorial
+
+In this tutorial, instead of walking you through how to build everything from
+scratch, we give you the code for how to make the maze game show above and
+your mission will be to make your own version of it.
+
+This workshop also, assumes that you have done the
+[Dodge](https://github.com/hackedu/hackedu/blob/master/workshops/dodge/README.md)
+workshop. Make sure you do that workshop first.
+
+## 3. Setup
+
+1. If you're more experienced and want to use your own editor, go ahead. Use the
+   below files to your editor. Do note however that the rest of this tutorial be
+   written assuming that you are using JS Bin.
+
+    - [this HTML file](https://gist.githubusercontent.com/jonleung/11ecd13a9d957003416b/raw/625081d81cc9c941007ddfd54c9ffdeec0782d62/index.html)
+    - [this JavaScript file](https://gist.githubusercontent.com/jonleung/11ecd13a9d957003416b/raw/625081d81cc9c941007ddfd54c9ffdeec0782d62/main.js)
+    - and [this CSS file](https://gist.githubusercontent.com/jonleung/11ecd13a9d957003416b/raw/625081d81cc9c941007ddfd54c9ffdeec0782d62/style.css)
+
+2. If you haven't already, make sure you have a JS Bin account first. <a
```

Comment:
`...make sure you have a JS Bin account first` -> `make sure you have created (and logged into) a JS Bin account first!`